catalyst/rl/offpolicy/scripts/run_trainer.py

```diff
@@ -51,8 +51,6 @@ def main(args, unknown_args):
         # come on, just refactor whole rl
         environment_fn = Registry.get_fn("environment", args.environment)
         env = environment_fn(**config["env"])
-        config["shared"]["observation_size"] = env.observation_shape[0]
```

Comment:
so...why have you delete them? (it's a legacy, nevertheless, I'm not sure if we could delete them easily)